client/my-sites/plan-price/index.jsx

```diff
@@ -24,13 +24,13 @@ export class PlanPrice extends Component {
 			translate,
 		} = this.props;
 
-		if ( ! currencyCode || ! ( typeof rawPrice === 'number' ) ) {
+		if ( ! currencyCode || rawPrice === undefined ) {
 			return null;
 		}
 
 		// "Normalize" the input price or price range.
 		const rawPriceRange = Array.isArray( rawPrice ) ? rawPrice.slice( 0, 2 ) : [ rawPrice ];
-		if ( rawPrice !== 0 && rawPriceRange.includes( 0 ) ) {
+		if ( rawPrice > 0 && rawPriceRange.includes( 0 ) ) {
```

Comment:
It occurs to me that `rawPrice > 0` when `rawPrice` is an array will not give us quite the right answer, probably. We might want to replace this with something different.

Reply:
Maybe:

```suggestion
		if ( Array.isArray( rawPrice ) && rawPriceRange.includes( 0 ) ) {
```